eng/scripts/Language-Settings.ps1

```diff
@@ -461,7 +461,14 @@ function GetExistingPackageVersions ($PackageName, $GroupId = $null)
   }
   catch
   {
-    LogError "Failed to retrieve package versions. `n$_"
+    if ($_.Exception.Response.StatusCode -eq 404) 
```

Comment:
Do we even want this to be warning given that this is an expected state?

Reply:
Ok, I see. So only error for exceptions that are not 404